tycho-plugins/target-platform-utils/src/main/java/org/jboss/tools/tycho/targets/TargetToRepoMojo.java

```diff
@@ -207,14 +226,14 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 		}
 	}
 
-    private static MirrorOptions createMirrorOptions() {
+    private MirrorOptions createMirrorOptions() {
         MirrorOptions options = new MirrorOptions();
-        options.setFollowOnlyFilteredRequirements(false);
-        options.setFollowStrictOnly(true);
-        options.setIncludeFeatures(true);
-        options.setIncludeNonGreedy(true);
-        options.setIncludeOptional(true);
-        options.setLatestVersionOnly(false);
+        options.setFollowOnlyFilteredRequirements(followOnlyFilteredRequirements);
```

Comment:
should these all be this.variable instead of just variable?

Reply:
I tend to prefer `this.variable` but wouldn't make it a blocker. That said, if we want to enforce it, we can set it in the .settings ;)